docs/gaia/gaia.rst

```diff
@@ -6,76 +6,80 @@
 Gaia TAP+ (`astroquery.gaia`)
 *****************************
 
-Gaia is an ambitious mission to chart a three-dimensional map of our Galaxy,
-the Milky Way, in the process revealing the composition, formation and evolution
-of the Galaxy. Gaia will provide unprecedented positional and radial velocity
-measurements with the accuracies needed to produce a stereoscopic and kinematic
-census of about one billion stars in our Galaxy and throughout the Local Group.
-This amounts to about 1 per cent of the Galactic stellar population.
+Gaia is a European space mission providing astrometry, photometry, and spectroscopy of
+more than 1000 million stars in the Milky Way. Also data for significant samples of
+extragalactic and Solar system objects is made available. The Gaia Archive contains
+deduced positions, parallaxes, proper motions, radial velocities, and brightnesses.
+Complementary information on multiplicity, photometric variability, and astrophysical
+parameters is provided for a large fraction of sources.
 
-If you use public Gaia DR1 data in your paper, please take note of our guide_ on
-how to acknowledge and cite Gaia DR1.
+If you use public Gaia data in your paper, please take note of our guide_ on
+how to acknowledge and cite Gaia data.
 
-.. _guide: https://gaia.esac.esa.int/documentation/GDR1/Miscellaneous/sec_credit_and_citation_instructions.html
+.. _guide: https://gea.esac.esa.int/archive/documentation/credits.html
```

Comment:
this reminds me @ceb8 and @keflavich (and @pllim if interested), that we better set up a linkchecker for astroquery as well, similarly to astropy core.

Reply:
Not a bad idea! But in my experience, some servers start to block the CI after a while (our own `stsci.edu` is one of them), so it might not catch everything but it is better than nothing!